.husky/pre-commit

```diff
@@ -1,8 +1,4 @@
 #!/bin/sh
-
-export NVM_DIR="$HOME/.nvm"
```

Comment:
@mariana-k I guess this change is not intentional?

Reply:
Reverted the file